calcom/cal.com
# style: Fix text wrapping issue in button

## What does this PR do?

This pull request resolves an issue where the button text wraps onto the next line in specific screen sizes. By applying the whitespace-nowrap utility class to the button element, the text now remains on a single line, even on screens with limited space. This enhancement ensures a consistent and visually pleasing user experience across various devices (look in the pictures provided for set up button).


### Before (Set Up Button)
![button-before](https://github.com/calcom/cal.com/assets/97639117/8f588a53-411a-4e9d-95df-76bd42d480d7)

### After  (Set Up Button)
![button-fixed](https://github.com/calcom/cal.com/assets/97639117/4c74e77a-a2cd-4b0c-87ee-a82dbefc23eb)




## Type of change



## How should this be tested?
- Ensure the whitespace-nowrap class is applied to the Button.
- Confirm the button text remains on a single line without wrapping  in all screen sizes.
- Check for any unintended layout or styling issues.


## Mandatory Tasks


## Checklist



- I have  read the [contributing guide](https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md)
- My code follow the style guidelines of this project
-  My PR doesn't needs changes to the documentation
